config.go

```diff
@@ -98,14 +107,21 @@ type LoggingConfig struct {
 
 // NewConfig returns a Config initialized with default values.
 func NewConfig() *Config {
+
+	defaultCachePath := "/tmp/trickster"
```

Comment:
This should be just `trickster.db`, and below we don't want to include an additional filename now.

Reply:
The defaultCachePath is used to instantiate both Filesystem (defaultCachePath) and BoltDb cache (defaultCachePath+filename) types, depending upon the supplied configuration. This change is as per your recommendation above to use a single configuration string to indicate the full path and use `filepath.Split()` to make the directory for a BoltDB cache.

I understand there is a philosophical difference here about whether the app should handle pathing and directory creation, but that disagreement is holding up the release. 

Here is how I propose we proceed. If, other than the disagreement about how Trickster should work with the filesystem, there are no issues with the PR, let's merge it. I would rather maintain consistency across all of the areas in the app where we touch the filesystem than introduce a new feature that works differently. That is what I am doing here.

If we really want to change the pathing philosophy across the entire application, let's work on that in a separate PR that addresses all of the filesystem-dependent areas of the app. That way, across any given release, how Trickster works with the filesystem is identical between features.

Reply:
I guess that's fine, I have a specific design idea I can propose.